wp-content/themes/csisnuclearnetwork/assets/_scss/components/_post-block-related.scss

```diff
@@ -38,6 +53,6 @@
     margin-top: rem(16);
     margin-bottom: rem(4);
```

Comment:
The margins should be on the element rather than the hover state.  Also, check the margins in Figma.